lib/atspi/atspi_node.cc

```diff
@@ -394,7 +394,8 @@ AtspiDocumentInterface AtspiNode::queryDocument() const {
 AtspiHyperlinkInterface AtspiNode::queryHyperlink() const {
   // Unlike the other interfaces, `atspi_accessible_get_hyperlink` gives us a
   // new hyperlink object that serves as the interface. Therefore we cannot
-  // free it until we're done with it. See also `atspi_hyperlink_interface.h`.
+  // free it until we're done with it, so we pass ownership to the
+  // AtspiHyperlinkInterface object. See also `atspi_hyperlink_interface.h`.
```

Comment:
These seems unrelated, and I see this line is removed in: https://github.com/Igalia/acacia/pull/210/files#diff-622c94f9ea8c0f81ccfd47ede4504ed3150af66b1377b72f129a1962361e9b0c

Reply:
Thanks for catching, will fix and land!